README.md

```diff
@@ -11,7 +11,7 @@
 - CSS
 
 ## Getting Started
-This project is to diplay a simple index html file with a gitignore file and linters check up.
+This project is to be a base for a portafolio: setup and mobile first.
 
 ## Authors
```

Comment:
- Please give more details in your read me about how to set up and run your project locally. You should assume anyone who doesn't know about coding is trying to use and run your project. So you should try to be as explicit and clear as you can 😄 😄

Reply:
Ok